Geometry/MTDNumberingBuilder/test/dd4hep_mtd_cfg.py

```diff
@@ -0,0 +1,48 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
+
+process = cms.Process("GeometryTest",dd4hep)
+
+process.source = cms.Source("EmptyIOVSource",
+                            lastValue = cms.uint64(1),
+                            timetype = cms.string('runnumber'),
+                            firstValue = cms.uint64(1),
+                            interval = cms.uint64(1)
+                            )
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1)
+)
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+process.MessageLogger.cerr.INFO.limit = -1
+
+
+process.DDDetectorESProducer = cms.ESSource("DDDetectorESProducer",
+                                            confGeomXMLFiles = cms.FileInPath('Geometry/MTDCommonData/data/dd4hep/cms-mtdD50-geometry.xml'),
+                                            appendToDataLabel = cms.string('')
+)
+
+process.DDCompactViewESProducer = cms.ESProducer("DDCompactViewESProducer",
+                                                 appendToDataLabel = cms.string('')
+)
+
+process.DDSpecParRegistryESProducer = cms.ESProducer("DDSpecParRegistryESProducer",
+                                                     appendToDataLabel = cms.string('')
+)
+
+# process.load("Geometry.MTDNumberingBuilder.mtdNumberingGeometry_cff")
```

Comment:
delete commented-out code (also below)

Reply:
@kpedro88 this will be done in next forthcoming PR where the corresponding code is added, here I have stripped the topology part only.